convert2rhel/data/8/x86_64/configs/centos-8-x86_64.cfg

```diff
@@ -5,17 +5,25 @@
 # Delimited by whitespace(s).
 gpg_fingerprints = 05b555b38483c65d
 
-# List of packages that have to be removed before the system conversion starts.
+# List of packages to be removed before the system conversion starts.
 # Delimited by any whitespace(s).
 excluded_pkgs =
   centos-logos*
   centos-indexhtml
-  centos*-release
-  centos*-repos
   centos-obsolete-packages
-  redhat-release*
   rhn*
   python3-rhn*
+  centos-linux-release
+
+# List of packages that contain repofiles from /etc/yum.repos.d/
+# Delimited by any whitespace(s).
+# 8.0: centos-release
+# 8.1-8.2: centos-repos
+# 8.3 and later: centos-linux-repos
+repofile_pkgs =
+  centos-release
+  centos-repos
+  centos-linux-repos
```

Comment:
You will need to deal with `centos-linux-release` and `centos-stream-release`.

Reply:
Do you mean `centos-release-stream` from http://mirror.centos.org/centos/8/extras/x86_64/os/Packages/?
Yeah, I should include that. Even though we don't plan on making the conversions of CentOS Stream to RHEL possible with this PR.

Reply:
I did actually mean `centos-stream-release`, but yeah, that one is important too.